test/icc-x-api/storage/storage.ts

```diff
@@ -0,0 +1,68 @@
+import {LocalStorageFacade} from "../../../icc-x-api/storage/LocalStorageFacade"
+import {expect} from "chai"
+
+const storage: Record<string, string> = {}
+
+class TestStorage implements LocalStorageFacade {
```

Comment:
If you do a specific implementation, then you test this test implementation : Use `LocalStorageImpl` instead.